File: webframework/src/main/java/com/brainz/framework/scheduling/repository/ScheduleTaskRepository.java
package com.brainz.framework.scheduling.repository;

import java.util.List;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import com.brainz.framework.scheduling.model.ScheduleTask;

@Repository
public interface ScheduleTaskRepository extends CrudRepository<ScheduleTask, Long> {
    List<ScheduleTask> findAll();
}

File: src/main/kotlin/co/brainz/framework/scheduling/repository/ScheduleTaskRepository.kt
package co.brainz.framework.scheduling.repository

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository;
import co.brainz.framework.scheduling.entity.ScheduleTaskEntity

@Repository
public interface ScheduleTaskRepository: CrudRepository<ScheduleTaskEntity, Long> {
    override fun findAll(): MutableList<ScheduleTaskEntity>
}
